src/scss/_media-queries.scss

```diff
@@ -12,11 +12,22 @@
 
 @media only screen and (max-width: 675px){
   header {
-    
+    display: flex;
+    flex-direction: column;
+    height: 8em;
+    line-height: 1em;
+    & h1 {
+      margin-top: 0.1em;
+      margin-bottom: 0.1em;
+    }
   }
 
   .poster-grid {
-    grid-template-columns: 1fr 1fr;
+    width: 100%;
+  }
+
+  .poster-img {
+    width: 200px;
```

Comment:
I think this is a good idea, this would of been my initial thought as well for doing responsiveness.